src/providers/oauth2.base.ts

```diff
@@ -1,5 +1,4 @@
-import type { EndpointOutput } from "@sveltejs/kit/types/endpoint";
-import { RequestEvent } from "@sveltejs/kit/types/hooks";
+import type { EndpointOutput, RequestEvent } from "@sveltejs/kit";
```

Comment:
Export renamed in https://github.com/sveltejs/kit/pull/4109 rename `EndpointOutput` to `RequestHandlerOutput` and tidy up